lib/actions/napster.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+function getMetadata(id, parentUri, type, title, cdudn) {
+  return `<DIDL-Lite xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:upnp="urn:schemas-upnp-org:metadata-1-0/upnp/"
+  xmlns:r="urn:schemas-rinconnetworks-com:metadata-1-0/" xmlns="urn:schemas-upnp-org:metadata-1-0/DIDL-Lite/">
+  <item id="${id}" parentID="${parentUri}" restricted="true"><dc:title>"${title}"</dc:title><upnp:class>${type}</upnp:class>
+  <desc id="cdudn" nameSpace="urn:schemas-rinconnetworks-com:metadata-1-0/">${cdudn}</desc></item></DIDL-Lite>`;
+}
+
+function getSongUri(id) {
+  return `x-sonos-http:song%3a${id}.mp4?sid=204&flags=8224&sn=5`;
```

Comment:
Napster has sid 203 on my system...? Could you check what id and type is returned from `/services/all` from the api?